firefox-ios/Client/Frontend/Toolbar+URLBar/TabLocationView.swift

```diff
@@ -1,516 +0,0 @@
-    private var themeManager: ThemeManager = AppContainer.shared.resolve()
-    let windowUUID: WindowUUID
-    let logger: Logger
-
-    var isTrackingProtectionDisplayed = false
-
-    /// Tracking protection button, gets updated from tabDidChangeContentBlocking
-    var blockerStatus: BlockerStatus = .noBlockedURLs {
-        didSet {
-            if oldValue != blockerStatus { setTrackingProtection(theme: currentTheme()) }
-        }
-    }
-
-    var hasSecureContent = false {
-        didSet {
-            if oldValue != hasSecureContent { setTrackingProtection(theme: currentTheme()) }
-        }
-    }
-
-    var url: URL? {
-        didSet {
-            hideButtons()
-            updateTextWithURL()
-            setNeedsUpdateConstraints()
-            showTrackingProtectionButton(for: url)
-        }
-    }
-
-    var readerModeState: ReaderModeState {
-        get {
-            return readerModeButton.readerModeState
-        }
-        set (newReaderModeState) {
-            guard newReaderModeState != self.readerModeButton.readerModeState else { return }
-            setReaderModeState(newReaderModeState)
-        }
-    }
-
-    lazy var contentView: UIStackView = {
-        let space1px = UIView.build()
-        space1px.widthAnchor.constraint(equalToConstant: 1).isActive = true
-        let subviews = [
-            trackingProtectionButton,
-            space1px,
-            urlTextField,
-            readerModeButton,
-            reloadButton
-        ]
-        let contentView = UIStackView(arrangedSubviews: subviews)
-        contentView.distribution = .fill
-        contentView.alignment = .center
-        contentView.translatesAutoresizingMaskIntoConstraints = false
-        return contentView
-    }()
-
-    lazy var urlTextField: URLTextField = .build { urlTextField in
-        // Prevent the field from compressing the toolbar buttons on the 4S in landscape.
-        urlTextField.setContentCompressionResistancePriority(UILayoutPriority(rawValue: 250), for: .horizontal)
-        urlTextField.accessibilityIdentifier = "url"
-        urlTextField.accessibilityActionsSource = self
-        urlTextField.backgroundColor = .clear
-        urlTextField.accessibilityLabel = .TabLocationAddressBarAccessibilityLabel
-        urlTextField.font = UIFont.preferredFont(forTextStyle: .body)
-        urlTextField.adjustsFontForContentSizeCategory = true
-
-        // Remove the default drop interaction from the URL text field so that our
-        // custom drop interaction on the BVC can accept dropped URLs.
-        if let dropInteraction = urlTextField.textDropInteraction {
-            urlTextField.removeInteraction(dropInteraction)
-        }
-    }
-
-    private func setURLTextfieldPlaceholder(theme: Theme) {
-        let attributes = [NSAttributedString.Key.foregroundColor: theme.colors.textSecondary]
-        urlTextField.attributedPlaceholder = NSAttributedString(
-            string: .TabLocationURLPlaceholder,
```

Comment:
That string is now only used in `AutocompleteTextField`, which I think can be removed too. So we can remove that class and deprecate that string as well!